### DISCUSSION TITLE
底部插槽部分可隐藏吗

### DISCUSSION THREAD

**Discussion by chaoyue1217** [opensumi/core#3294] [Q&A] [answered]
![image](https://github.com/opensumi/core/assets/18662948/29db467e-8f7d-47f3-aaf1-574090a88352)
如图红框中的部分是一直在界面上显示的，这部分有办法隐藏吗

**Comment by bytemain** [comment, marked as answer]
这部分我们提供了一个配置进行隐藏，
你可以在程序的默认配置中设置该项（https://github.com/opensumi/core/blob/0de5b851f0f6385aa814c7540d75445b3ebd7df7/packages/startup/entry/web/app.tsx#L42 ）：
```json
{
  "workbench.hideSlotTabBarWhenHidePanel": true
}
```
